### DISCUSSION TITLE
collaborator_times_per_round meaning in aggregator function

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#164] [General]
Hi 
what does collaborator_times_per_round mean in the aggregator function?

e.g output of collaborator_times_per_round 
`collaborator_times_per_round: {0: {'1': 5297.622019700235, '10': 923.1055982064671, '11': 258.42950340352223, '12': 762.7787493043197, '13': 458.62167467356915, '14': 626.9453080987761, '15': 211.85548269835772, '16': 227.0132381973164, '17': 45.36113136706723, '18': 240.12554785139815, '19': 87.11048716924347, '2': 560.7956428816875, '20': 119.31800027988054, '21': 78.33730623719993, '22': 201.37608453705639, '23': 113.12383611248681, '24': 1054.0792870729863, '25': 426.62727832315227, '26': 355.48413050267175, '27': 234.4789082724287, '28': 248.43012712767842, '29': 66.75629262281399, '3': 1381.2864506190942, '30': 287.3511903994488, '31': 91.3601468865659, '32': 115.59048182447027, '33': 245.73217476668356, '4': 257.0524523434632, '5': 90.38409338507475, '6': 178.84964220472395, '7': 122.00833415917701, '8': 80.00203310661163, '9': 131.13575852719356}}`

**Comment by brandon-edwards** [comment]
This provides info regarding how log it took each collaborator to perform all of its round tasks for each round.
